stdlib/printexc.mli

```diff
@@ -193,7 +193,7 @@ val backtrace_slots : raw_backtrace -> backtrace_slot array option
     recent function call, raise, or primitive [get_backtrace] call in
     the trace.
 
-    Some possible reasons for returning [None] are as follow:
+    Some possible reasons for returning [None] are as follows:
```

Comment:
Well, this has sparked an amusing exchange with a good friend in my other line of business (musicians are pedants beyond a computer scientist/programmer's wildest dreams...). I believe Americans would always put "as follows", which should give us clear guidance as to which is elegant! :wink:

You would not put "Some possible reasons for returning [None] follows:", however, brevity is King, and it would be much simpler just to delete "as follow" entirely and have "Some possible reasons for returning [None] are:"